pkg/kubecfg/eval.go

```diff
@@ -26,7 +27,21 @@ func (c EvalCmd) Run(ctx context.Context, vm *jsonnet.VM, path string) error {
 	if err != nil {
 		return err
 	}
-	eval := fmt.Sprintf("((import %q) { Z____expr__Z_:: %s}).Z____expr__Z_", pathURL, expr)
+	var eval string
+	if err != nil {
+		return err
+	}
+	if len(tla) > 0 {
+		formals := strings.Join(tla, ",")
+		pairs := make([]string, len(tla))
+		for i := range tla {
+			pairs[i] = fmt.Sprintf("%s=%s", tla[i], tla[i])
```

Comment:
i don't really follow exactly what's happening here but this looks suspicious to me:

`fmt.Sprintf("%s=%s", tla[i], tla[i])`

should it be the same value either side of the equals sign?

Reply:
added a comment in the code

Reply:
basically the trick is `function(foo,bar) (import %q)(foo=foo,bar=bar)`